huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_cassandra_instance.go

```diff
@@ -258,7 +265,17 @@ func ResourceGeminiDBInstanceV3() *schema.Resource {
 	}
 }
 
-func resourceGeminiDBDataStore(d *schema.ResourceData) instances.DataStore {
+func resourceGaussDBCassandraInstanceCreate(d *schema.ResourceData, meta interface{}) error {
+	defaults := defaultValues{
+		Mode:      "Cluster",
+		dbType:    "cassandra",
+		dbVersion: "3.11",
```

Comment:
seem all params are required, the default values are unnecessary

Reply:
this is correct under current logic
will refactor the gaussdb resources and use the true default value way latter